### PR TITLE
Split "Recommended Software for Computer Science" section from First Year Guide

### PR DESCRIPTION
**Pull request by angelohoeung**
### What are you trying to accomplish?
Separating the "Recommended Software for Computer Science" section from the first year guide, and linking to it in the first year guide.
### How are you accomplishing it?
Cutting and pasting the "Recommended Software for Computer Science" section to a new file, linking it in the first year guide, and making required changes to index.
### Is there anything reviewers should know?
Closes issue #177.
### Checks before you create your pull request

-   [x] Did you read and follow article requirements?
-   [x] Did you run prettier?
-   [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: resources/guides/recommended-software.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+As a computer science student, you have access to the School of Computer Science's servers, which provide software in a Linux environment that you are free to use. On campus, they can be accessed using School of Computer Science computer labs. Remotely, they can be accessed in several ways.
+
+-   SSH (recommended): Remotely use a shell on the server.
+-   NoMachine: Remote desktop environment.
+
+:::important
+
+-   You must [authenticate your UWin account](https://auth.cs.uwindsor.ca/) with the School of Computer Science before you can log in remotely.
+-   To log in remotely from off-campus, you must be on the VPN first. Please refer to our guide on how to set up the VPN [here](https://uwindsorcss.github.io/wiki/resources/guides/vpn).
```
Comment (tanzimfh) [inline resources/guides/recommended-software.md]:
```suggestion
-   To log in remotely from off-campus, you must be on the VPN first. Refer to the [VPN Setup Guide](./vpn).
```

File: resources/guides/first-year-guide.md [hunk 2]
```diff
@@ -49,176 +49,4 @@ Incoming first year students typically have the earliest **appointment time** an
...
-```
-
-For example, if chipcoolcat@uwindsor.ca wanted to connect, they would use the following command.
-
-```
-ssh chipcoolcat@cs.uwindsor.ca
-```
-
-Use your UWin Account password as the password for logging in once prompted.
+For a detailed guide on the recommended software for computer science students, see the [Recommended Software for Computer Science](recommended-software.md) article.
```
Comment (tanzimfh) [inline resources/guides/first-year-guide.md]:
```suggestion
For a detailed guide on the recommended software for computer science students, see [Recommended Software](recommended-software.md).
```
more concise

**Review by tanzimfh — COMMENTED** [review]
I prefer just `Recommended Software` for the title. WDYT?

File: resources/guides/first-year-guide.md (hunk 2 above)
Comment (angelohoeung) [reply resources/guides/first-year-guide.md]: Yeah, I agree.

**Review by BorhanSaflo — APPROVED** [review]
LGTM

**Pull request merged**